docs/color.md

```diff
@@ -0,0 +1,178 @@
+Found in `lua/pixelui/core/cl_color.lua`
+
+# PIXEL.DecToHex
+
+```lua
+function PIXEL.DecToHex(dec, zeros)
+```
+
+Converts a decimal value to a hexadecimal string representation.
+
+**Parameters:**
+- `dec` (number) - The decimal value to convert.
+- `zeros` (number, optional) - The number of leading zeros to add to the hexadecimal string. Defaults to 2.
+
+**Returns:**
+- `hex` (string) - The hexadecimal string representation of the decimal value.
+
+---
+
+# PIXEL.ColorToHex
+
+```lua
+function PIXEL.ColorToHex(color)
+```
+
+Converts a color to its hexadecimal string representation.
+
+**Parameters:**
+- `color` (Color) - The color to convert.
+
+**Returns:**
+- `hex` (string) - The hexadecimal string representation of the color.
+
+---
+
+# PIXEL.ColorToHSL
+
+```lua
+function PIXEL.ColorToHSL(color)
+```
+
+Converts a color to its HSL (Hue, Saturation, Lightness) representation.
+
+**Parameters:**
+- `color` (Color) - The color to convert.
+
+**Returns:**
+- `h` (number) - The hue value of the color.
+- `s` (number) - The saturation value of the color.
+- `l` (number) - The lightness value of the color.
+
+---
+
+# PIXEL.HSLToColor
+
+```lua
+function PIXEL.HSLToColor(h, s, l, a)
+```
+
+Converts HSL (Hue, Saturation, Lightness) values to a color.
+
+**Parameters:**
+- `h` (number) - The hue value.
```

Comment:
It would be better to use ordered lists for method parameters. The same can be done for returns that have multiple values.